### ISSUE TITLE
Idea: have the bot periodically check the channel order and sort the courses if they are out of order

### ISSUE THREAD

**Issue by DenverCoder1** [DenverCoder1/jct-discord-bot#209]

**Comment by abrahammurciano** [comment]
Good idea. I'd like to add, that when someone does reorder them, it should be logged in the bot logs so we can check who did it and strip them of their admin role

**Comment by abrahammurciano** [comment]
Better yet, do it on channel reorder instead of periodically

**Comment by DenverCoder1** [comment]
on_guild_channel_update gets triggered when a channel's position is updated.

The only parameters are "before" and "after".

I don't think it's possible to know who changed it.
